### ISSUE TITLE
Question about the performance on CIFAR100

### ISSUE THREAD

**Issue by GWwangshuo** [sinhasam/vaal#16]
Hi, thanks for your interesting work. I found your provided code can give the similar performance in your paper on CIFAR10. However, this implementation gives only 5%-10% accuracy on CIFAR100 test set when training with 5000 seed set. Could your please give me some help to handle this problem? Thanks.

Moreover, I read your implementation carefully and found there is no normalization in your dataloader? Why? Thanks again.

**Comment by sinhasam** [comment]
Thanks for bringing it to my attention. This should hopefully be fixed. Please email if it is not.
